app/views/providers/show.html.erb

```diff
@@ -44,8 +44,8 @@
       <% if @provider_view.show_notifications_link? %>
         <%= render partial: "providers/notifications_sign_up_link" %>
       <% end %>
-      <h2 class="govuk-heading-s govuk-!-margin-bottom-2"><%= govuk_link_to "Request access", request_access_provider_path(code: @provider.provider_code) %></h2>
-      <p class="govuk-body">Request a DfE Sign-in account for others who manage your courses.</p>
+      <h2 class="govuk-heading-s govuk-!-margin-bottom-2"><%= govuk_link_to "Users", users_provider_path(code: @provider.provider_code) %></h2>
+      <p class="govuk-body">View users who manage your courses.</p>
```

Comment:
Should we not mention that they can also invite users from that page otherwise they may think that they cant add users without contacting support.

Reply:
Paul G has signed this off. We can improve at a later date.